model-mapper/src/com/baeldung/util/UserPropertyMap.java

```diff
@@ -0,0 +1,36 @@
+package com.baeldung.util;
+
+import com.baeldung.model.User;
+import com.baeldung.model.UserDTO;
+import com.baeldung.model.UserList;
+import org.modelmapper.AbstractConverter;
+import org.modelmapper.Converter;
+import org.modelmapper.PropertyMap;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * @author sasam0320
```

Comment:
describe and use put your name and remove the date (I am not even sure that's a valid Javadoc tag)